server/models/index.js

```diff
@@ -78,6 +79,8 @@ function setupModels(client) {
   m.User.hasMany(m.Activity);
 
   m.Activity.belongsTo(m.Transaction);
+  m.Activity.belongsTo(m.Expense);
```

Comment:
I don't think we need to add `Expense` field in `Activity` model. Would rather keep everything in the `data` field.